### PR TITLE
Adding ZBJetsToNuNu and ZJetsToNuNu samples

### PR DESCRIPTION
**Pull request by tiwariPC**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJetsToNuNu/ZJetsToNuNu_Zpt-100to200_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import model sm-ckm_no_b_mass
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > ve ve~ / h @0
+add process p p > ve ve~ j / h @1
+add process p p > ve ve~ j j / h @2
+add process p p > ve ve~ j j j / h @3
+add process p p > ve ve~ j j j j / h @4
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJetsToNuNu/ZJetsToNuNu_Zpt-100to200_proc_card.dat:8]: Is there a reason to only do "ve ve~" for ZJets while you do "vl vl~" for ZJets+b?

Comment (tiwariPC) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJetsToNuNu/ZJetsToNuNu_Zpt-100to200_proc_card.dat]: Thanks, @sihyunjeon for noticing this, I have fixed this bug now

**Comment by agrohsje** [comment]
cards look good to me. what is the reason to have /h in the process line?

**Comment by gourangakole** [comment]
> cards look good to me. what is the reason to have /h in the process line?

Hi @agrohsje , I agree with your point. To my understanding, 
in SM Higgs has no direct coupling with the neutrinos, so it should not matter 
but better to remove. 

thank you for catching that.

**Comment by agrohsje** [comment]
@tiwariPC : can we quickly fix and then move forward.
@sihyunjeon are you ok with merging?

**Comment by sihyunjeon** [comment]
@agrohsje Yes.
The higgs seems to be a null command but let's just wait for @tiwariPC to comment. After this, should be merged.

**Comment by tiwariPC** [comment]
Hi @agrohsje  and @sihyunjeon, 
I can remove that part, ```/ h @1``` this whole part shall be removed right?
My second question is that, I made the grid packs with these current cards, as it seems the \h command is null, so shall I remake them or they are okay?

**Comment by sihyunjeon** [comment]
You can keep the @1 @2 ... just remove "/ h".
And I am fine with the current gridpacks though they have "/ h" since the physics is the same.
But @agrohsje will let you know what to do.

**Comment by agrohsje** [comment]
@tiwariPC it is fine to have it in the gridpacks. i would just remove it here in the pr to keep it clean and meaningful.

**Comment by tiwariPC** [comment]
Okay Thanks @agrohsje, I have removed them here

**Pull request merged**